hack/ci-deploy-release.sh

```diff
@@ -78,8 +78,17 @@ else
  kubectl_create -f="${SO_NODECONFIG_PATH}"
 fi
 
-kubectl_create -n=local-csi-driver -f="${source_url}/${revision}/examples/common/local-volume-provisioner/local-csi-driver/"{00_namespace.yaml,00_scylladb-local-xfs.storageclass.yaml,10_csidriver.yaml,10_driver.serviceaccount.yaml,10_provisioner_clusterrole.yaml,20_provisioner_clusterrolebinding.yaml,50_daemonset.yaml}
-kubectl -n=local-csi-driver rollout status --timeout=5m daemonset.apps/local-csi-driver
+if [[ -z "${SO_CSI_DRIVER_PATH:-}" && -n "${SO_CSI_DRIVER_PATH+x}" ]]; then
+  echo "Skipping CSI driver creation"
+else
+  if [[ -z "${SO_CSI_DRIVER_PATH+x}" ]]; then
```

Comment:
i think you actually need just 2 comparisons
```
if [[ -z "${SO_CSI_DRIVER_PATH+x}" ]]; then
...
else if [[ -n "${SO_CSI_DRIVER_PATH}" ]]; then
...
else
...
fi
```

Reply:
that's just a tradeoff between the nested check and duplicating the waiting for rollout line

Reply:
it avoiding testing the same condition `"${SO_CSI_DRIVER_PATH+x}"` twice in the same if block, so the branching is easier to read and evaluate

Reply:
I don't mind either way, changed